apps/rotm/index.js

```diff
@@ -135,5 +136,6 @@ module.exports = {
       backLink: false
     },
     '/exit': {}
-  }
+  },
+  behaviours: [reCaptcha]
```

Comment:
By addin the behaviour here, will it be used in every lifecycle of HOF e.g. get etc?  I can't remember?
Would it be better to add it in the submission page?